cli/command/utils.go

```diff
@@ -115,5 +117,23 @@ func PruneFilters(dockerCli Cli, pruneFilters filters.Args) filters.Args {
 		pruneFilters.Add(parts[0], parts[1])
 	}
 
+	// Server APIs older than 1.29 will just ignore label filters.
```

Comment:
I feel we should do that server side though (the `cli` should be *as dumb as possible*)
cc @thaJeztah @dnephin